sdk/vaa/structs.go

```diff
@@ -563,9 +563,12 @@ const (
 	minHeadlessVAALength = 51 // HEADER
 	minVAALength         = 57 // HEADER + BODY
 
-	SupportedVAAVersion = 0x01
+	VaaVersion1   = 0x01
```

Comment:
It's weird to have VaaVersion 1 (simple numbering) vs TSSVaaVersion (a qualitative name). 
Maybe instead of VaaVersion1, call it MultisigVaaVersion